CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+**2.6.0** (5 Aug 2015)
+- add `time_event` method
+- update snippet to load mixpanel correctly from cdn when ﬁle is opened locally
```

Comment:
I wouldn't want to forbid utf8 in HTML files. I would argue that utf8 in `.py` files is not the end of the world either, so long as there's a corresponding `coding: utf-8` comment. I'd be down to warn on utf-8 in `.txt`/`.md` though ;)